rest/blip_api_crud_test.go

```diff
@@ -2040,8 +2035,7 @@ func TestRemovedMessageWithAlternateAccess(t *testing.T) {
 
 		err = btcRunner.StartOneshotPull(btc.id)
 		assert.NoError(t, err)
-		_, ok := btcRunner.WaitForVersion(btc.id, docID, version)
-		assert.True(t, ok)
+		_ = btcRunner.WaitForVersion(btc.id, docID, version)
```

Comment:
Nitpick: Same as above in this file too?

Reply:
WaitForMessage return values are used in blip_api_delta_sync_test.go